msal/tests/Test.MSAL.NET.Unit.net45/CoreTests/InstanceTests/AadAuthorityTests.cs

```diff
@@ -304,6 +300,8 @@ public void FailedTenantDiscoveryMissingEndpointsTest()
         [TestCategory("AadAuthorityTests")]
         public void CanonicalAuthorityInitTest()
         {
+            var serviceBundle = ServiceBundle.CreateForProduction();
```

Comment:
>CreateForProduction [](start = 46, length = 19)

is this on purpose?

Reply:
Yes, this is on purpose.  CreateForTest takes in a MockHttpManager.  This doesn't have/need one so we just create a Production servicebundle.  I could change this name to CreateDefault and CreateWithMockHttp if that makes this usage clearer.